airflow/cli/commands/role_command.py

```diff
@@ -47,6 +47,16 @@ def roles_create(args):
     print(f"Added {len(args.role)} role(s)")
 
 
+@cli_utils.action_cli
+@suppress_logs_and_warning
+def roles_delete(args):
+    """Deletes role in DB"""
+    appbuilder = cached_app().appbuilder
+    for role_name in args.role:
+        appbuilder.sm.delete_role(role_name)
```

Comment:
This crashes if `role_name` does not exist so you need to validate before passing the value in.